src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationModalMap.js

```diff
@@ -551,15 +552,21 @@ const ImageAnnotationModalMap = ({
         }}
       />
       {hasMapLoaded ? (
-        <MapResetButton type="button" onClick={resetZoom}>
-          <IconReset />
+        <MapResetButton type="button" onClick={resetZoom} title="reset zoom">
+          <Tooltip tooltipText="Reset Zoom" id="reset-zoom" $position="right">
```

Comment:
Does tooltip text need to be in the language file?

Reply:
No, we've started cutting that corner. Its been argued that its easy enough to go through later and move stuff to a language file when we cross that bridge.